### PR TITLE
feat: add about us section

### PR DESCRIPTION
**Pull request by divyanshugarg36**
- add new hook for body scroll on modal open
- add modal component
- add new cross icon

### CODE REVIEW

**Comment by divyanshugarg36** [comment]
Fixes #19

File: src/components/AboutUs.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      fullTitle: t('ABOUT.ABOUT_3.FULL_TITLE'),
+      description: t('ABOUT.ABOUT_3.DESCRIPTION'),
+    },
+    {
+      image: about4,
+      title: t('ABOUT.ABOUT_4.TITLE'),
+      fullTitle: t('ABOUT.ABOUT_4.FULL_TITLE'),
+      description: t('ABOUT.ABOUT_4.DESCRIPTION'),
+    },
+  ];
```
Comment (theamanjs) [inline src/components/AboutUs.tsx]: Can this static data be moved to different file?

File: src/i18n/locales/en.json [hunk 2]
```diff
@@ -52,6 +52,29 @@
       "DESCRIPTION": "BRL-CAD respects your freedom so our code is open source under OSI approved license terms, which means you can customize it according to your needs.It also means that you will get this software Free of cost and we won't charge you ever for any update or support."
```
Comment (theamanjs) [inline src/i18n/locales/en.json:52]: I am just asking @divyanshugarg36 where you are fetching this content from?

File: src/styles/common/aboutBlock.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    width: 100%;
+  }
+
+  h1 {
+    @include flex($direction: column, $justifyContent: center);
+    cursor: pointer;
+    font-weight: normal;
+    font-size: 16px;
+    color: var(--dark-grey);
+    text-align: center;
```
Comment (theamanjs) [inline src/styles/common/aboutBlock.scss:23]: Didn't you create a mixin for font-related properties?

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en.json (hunk 2 above)
Comment (divyanshugarg36) [reply src/i18n/locales/en.json:52]: Mostly from original site and wikipedia

File: src/styles/common/aboutBlock.scss (hunk 3 above)
Comment (divyanshugarg36) [reply src/styles/common/aboutBlock.scss:23]: We do have an issue listed for this one but I will be sending a PR soon

File: src/components/AboutUs.tsx (hunk 1 above)
Comment (divyanshugarg36) [reply src/components/AboutUs.tsx]: This is fixed

File: src/styles/common/aboutBlock.scss (hunk 3 above)
Comment (theamanjs) [reply src/styles/common/aboutBlock.scss:23]: okay

File: src/i18n/locales/en.json (hunk 2 above)
Comment (theamanjs) [reply src/i18n/locales/en.json:52]: okay great

**Review by theamanjs — APPROVED** [review]

**Pull request merged**